apiary.apib

```diff
@@ -542,7 +542,183 @@ API endpoints that Operation Code's Rails backend makes available to its React f
             errors: "Some error message"
         }
 
+## Mentor Request | Create [/api/v1/requests{?user_id,details,language,service_id,requested_mentor_id,status,assigned_mentor_id}]
++ Parameters
+    
+    + user_id (integer, required) - Identification of the User requesting a mentor in the form of an integer
+    + details (string, optional) - Details of the mentor request in the form of a string
+    + language (string, optional) - Primary language of the requesting mentee in the form of a string
+    + service_id (integer, optional) - Indentification of the mentorship request in the form of an integer
+    + requested_mentor_id (integer, optional) - Requested mentor's id in the form of an integer
+    + status (string, optional) - Current fullfillment status in the form on a string
+    + assigned_mentor_id (integer, optional) - Assigned mentor'd id in the form of an integer
+    
+    
+### Create a New Mentor Request [POST]
++ Request (application/json)
+
+    + Headers
+    
+            Authorization: Bearer Access-Token
+
+    + Body
+
+            {
+                "user_id": 4600,
+                "details": "I want to type well",
+                "language": "English",
+                "service_id": 200,
+                "requested_mentor_id": 250,
+                "status": "Open",
+                "assigned_mentor_id": 250
+            }
+
++ Response 201 (application/json)
+
+    + Body
+
+            {
+                "user_id": 4600,
+                "details": "I want to type well",
+                "language": "English",
+                "service_id": 200,
+                "requested_mentor_id": 250,
+                "status": "Open",
+                "assigned_mentor_id": 250
+            }
+
++ Response 422 (application/json)
+
+    + Body
+
+            {
+                errors: "Some error message"
+            }
+
+## Mentor Request | Show [/api/v1/requests{?user_id}]
++ Parameters
+    
+    + user_id (integer, required) - Identification of the User requesting a mentor in the form of an integer
+
+    
+### Show an Exiting Request From User ID [POST]
+
++ Request (application/json)
+
+    + Headers
+
+            Authorization: Bearer Access-Token
+
+    + Body
 
+            {
+                "user_id": 5200
+
+            }
+
++ Response 200 (application/json)
+
+    + Body
+
+            {
+                "user_id": 5200,
+                "details": "I want to program",
+                "language": "English",
+                "service_id": 210,
+                "requested_mentor_id": 260,
+                "status": "Closed",
+                "assigned_mentor_id": 260
+            }
+
+
+## Mentor Request | Update [/api/v1/requests{?user_id,details,language,service_id,requested_mentor_id,status,assigned_mentor_id}]
```

Comment:
The `update` endpoint takes the `request_id` as a part of the url itself, not a parameter.

Here is an example from [TeamMember#update](https://github.com/OperationCode/operationcode_backend/blob/master/apiary.apib#L896).